docs/source/tagset/elemento-ref-list.rst

```diff
@@ -3,21 +3,20 @@
 <ref-list>
 ----------
 
-Aparece em
+Aparece em:
+
   :ref:`elemento-back`
 
-Ocorre
+Ocorre:
+
   Zero ou mais vezes
 
-  O conjunto de referências biliográficas de um artigo é representado pela tag
-  ``<ref-list>``. Esse elemento deve conter, obrigatoriamente, três tags: 
-  :ref:`elemento-ref`, :ref:`elemento-mixed-citation` e
-  :ref:`elemento-element-citation`.
+Representa o conjunto de referências bibliográficas de um artigo e deve conter, obrigatoriamente, três elementos: :ref:`elemento-ref`, :ref:`elemento-mixed-citation` e :ref:`elemento-element-citation`.
```

Comment:
@gustavofonseca, o correto da hierarquia é como então? seria algo como:

``<ref-list>``
     ``<ref>``
          ``<element-citation>``
          ``<mixed- citation>``

é isso? Se for, poderia ser alterado para:

Representa o conjunto de referências bibliográficas de um artigo e deve conter, obrigatoriamente, o elemento :ref:`elemento-ref`, que por sua vez contém :ref:`elemento-mixed-citation` e :ref:`elemento-element-citation`.

Gostaria apenas de lembrar que minha leitura (e eventuais ajustes) são a interpretação do que vocês escreveram, pois sou leigo no assunto (SciELO PS).